docs/release-notes/8.2.asciidoc

```diff
@@ -41,6 +41,7 @@
 ==== Known issues
 * On macOS versions before 12.4, if {elastic-endpoint} is used with other products that monitor or manage network traffic (such as antivirus programs, firewalls, or VPNs), users might experience network connection issues. To resolve this issue, upgrade to macOS 12.4 or later.
 * Indicator match rules cannot use the `.items-*` system index and will encounter execution errors when run. Avoid using indices populated from value lists for indicator match rules ({pull}133457[#133457]).
+* The `matches` operator is not supported for exceptions to detection rules ({issue}136340[#136340]).
```

Comment:
Per sync discussion, we can add - "If you encounter a rule execution failure, the rule can be restored by deleting the exception item."